docs/services/gcp-services-guide.md

```diff
@@ -0,0 +1,536 @@
+Spec:
+  Class Ref:
+    Name:          mysql-standard
+  Engine Version:  5.6
+  Resource Ref:
+    API Version:  database.gcp.crossplane.io/v1alpha2
+    Kind:         CloudsqlInstance
+    Name:         mysqlinstance-6a7fe064-d888-11e9-ab90-42b6bb22213a
+    Namespace:    gcp-infra-dev
+  Write Connection Secret To Ref:
+    Name:  wordpressmysql
+Status:
+  Conditions:
+    Last Transition Time:  2019-09-16T13:46:42Z
+    Reason:                Managed claim is waiting for managed resource to become bindable
+    Status:                False
+    Type:                  Ready
+    Last Transition Time:  2019-09-16T13:46:42Z
+    Reason:                Successfully reconciled managed resource
+    Status:                True
+    Type:                  Synced
+Events:                    <none>
+```
+
+*Note: You must wait until the claim becomes bound before continuing with this
+guide. It could take a few minutes for GCP to complete CloudSQL creation.*
+
+#### Connecting MySQL Instance and GKE Cluster
+Before we install Wordpress, we need establish connectivity between our MySQL
+database and our GKE cluster. We can do this by creating a [Private Service 
+Connection](https://cloud.google.com/vpc/docs/configure-private-services-access).
+
+You can create it following the link above or use Crossplane to do it:
+
+* Create a `GlobalAddress` and `Connection` resources:
+```bash
+cat > network.yaml <<EOF
+---
+# example-globaladdress defines the IP range that will be allocated for cloud services connecting
+# to the instances in the given Network.
+apiVersion: compute.gcp.crossplane.io/v1alpha2
+kind: GlobalAddress
+metadata:
+  name: example-globaladdress
+  namespace: $INFRA_NAMESPACE
+spec:
+  providerRef:
+    name: gcp-provider
+    namespace: $INFRA_NAMESPACE
+  reclaimPolicy: Delete
+  name: example-globaladdress
+  purpose: VPC_PEERING
+  addressType: INTERNAL
+  prefixLength: 16
+  network: projects/$PROJECT_ID/global/networks/$NETWORK_NAME
+---
+# example-connection is what allows cloud services to use the allocated GlobalAddress for communication. Behind
+# the scenes, it creates a VPC peering to the network that those service instances actually live.
+apiVersion: servicenetworking.gcp.crossplane.io/v1alpha2
+kind: Connection
+metadata:
+  name: example-connection
+  namespace: $INFRA_NAMESPACE
+spec:
+  providerRef:
+    name: gcp-provider
+    namespace: $INFRA_NAMESPACE
+  reclaimPolicy: Delete
+  parent: services/servicenetworking.googleapis.com
+  network: projects/$PROJECT_ID/global/networks/$NETWORK_NAME
+  reservedPeeringRanges:
+    - example-globaladdress
+EOF
+
+kubectl apply -f network.yaml
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl -n $INFRA_NAMESPACE get connection example-connection -o custom-columns='NAME:.metadata.name,FIRST_CONDITION:.status.conditions[0].status,SECOND_CONDITION:.status.conditions[1].status'
```

Comment:
What are these conditions? Are there names we can use to explain them, instead of first and second?

Reply:
They have names but it's an array and no one knows the order of the conditions at the time of kubectl command execution.

Reply:
Dang